{
	"documenttitle": {
		"text": "Red Hat Security Advisory: mariadb:10.3 security and bug fix update",
		"lang": "en"
	},
	"DocumentType": "Security Advisory",
	"documentpublisher": {
		"type": "Vendor",
		"ContactDetails": "secalert@redhat.com",
		"IssuingAuthority": "Red Hat Product Security"
	},
	"documenttracking": {
		"identification": {
			"ID": "RHSA-2022:4818"
		},
		"Status": "Final",
		"Version": "1",
		"revisionhistory": {
			"revision": {
				"Number": "1",
				"Date": "2022-06-07T09:30:00Z",
				"Description": "Current version"
			}
		},
		"InitialReleaseDate": "2022-05-31T12:20:00Z",
		"CurrentReleaseDate": "2022-06-07T09:30:00Z",
		"generator": {
			"Engine": "Red Hat SDEngine 3.7.0",
			"Date": "2022-06-07T10:28:00Z"
		}
	},
	"documentnotes": {
		"note": [
			{
				"text": "\nAn update for the mariadb:10.3 module is now available for Red Hat Enterprise Linux 8.4 Extended Update Support.\n\nRed Hat Product Security has rated this update as having a security impact of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.\n    ",
				"title": "Topic",
				"type": "Summary",
				"ordinal": "1",
				"lang": "en"
			},
			{
				"text": "\nMariaDB is a multi-user, multi-threaded SQL database server that is binary compatible with MySQL. \n\nThe following packages have been upgraded to a later upstream version: mariadb (10.3.32), galera (25.3.34).\n\nSecurity Fix(es):\n\n* mysql: Server: DML unspecified vulnerability (CPU Apr 2021) (CVE-2021-2154)\n\n* mysql: Server: DML unspecified vulnerability (CPU Apr 2021) (CVE-2021-2166)\n\n* mysql: InnoDB unspecified vulnerability (CPU Jul 2021) (CVE-2021-2372)\n\n* mysql: InnoDB unspecified vulnerability (CPU Jul 2021) (CVE-2021-2389)\n\n* mysql: InnoDB unspecified vulnerability (CPU Oct 2021) (CVE-2021-35604)\n\n* mariadb: Integer overflow in sql_lex.cc integer leading to crash (CVE-2021-46667)\n\n* mariadb: crash in Used_tables_and_const_cache::used_tables_and_const_cache_join (CVE-2022-27385)\n\n* mariadb: Crash in get_sort_by_table() in subquery with ORDER BY having outer ref (CVE-2021-46657)\n\n* mariadb: save_window_function_values triggers an abort during IN subquery (CVE-2021-46658)\n\n* mariadb: Crash in set_var.cc via certain UPDATE queries with nested subqueries (CVE-2021-46662)\n\n* mariadb: Crash caused by mishandling of a pushdown from a HAVING clause to a WHERE clause (CVE-2021-46666)\n\n* mariadb: No password masking in audit log when using ALTER USER <user> IDENTIFIED BY <password> command (BZ#1981332)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nBug Fix(es):\n\n* WSREP race condition causes crash in mariadb 10.3.28, fixed in 10.3.32 (BZ#2077509)\n\n* mariadb:10.3/mariadb: /etc/security/user_map.conf getting overwritten with mariadb-server upgrade (BZ#2079855)\n\n* Galera doesn't work without 'procps-ng' package MariaDB-10.3 (BZ#2079858)\n\n* Tracker: Rebase galera package to the newest for MariaDB-10.3 (25.3.34) (BZ#2079859)\n\n* MariaDB logrotate leads to \"gzip: stdin: file size changed while zipping\" (BZ#2080159)\n    ",
				"title": "Details",
				"type": "General",
				"ordinal": "2",
				"lang": "en"
			},
			{
				"text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
				"title": "Terms of Use",
				"type": "Legal Disclaimer",
				"ordinal": "3",
				"lang": "en"
			}
		]
	},
	"documentdistribution": {
		"text": "Copyright Â© 2022 Red Hat, Inc. All rights reserved.",
		"lang": "en"
	},
	"aggregateseverity": {
		"text": "Moderate",
		"namespace": "https://access.redhat.com/security/updates/classification/"
	},
	"documentreferences": {
		"reference": [
			{
				"type": "Self",
				"URL": "https://access.redhat.com/errata/RHSA-2022:4818",
				"Description": "https://access.redhat.com/errata/RHSA-2022:4818"
			},
			{
				"type": "External",
				"URL": "https://access.redhat.com/security/updates/classification/#moderate",
				"Description": "https://access.redhat.com/security/updates/classification/#moderate"
			}
		]
	},
	"vulnerability": [
		{
			"ordinal": "1",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mysql: Server: DML unspecified vulnerability (CPU Apr 2021)",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-04-20T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-2154",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.oracle.com/security-alerts/cpuapr2021.html#AppendixMSQL",
						"Description": "https://www.oracle.com/security-alerts/cpuapr2021.html#AppendixMSQL"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-2154",
						"Description": "CVE-2021-2154"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1951752",
						"Description": "bz#1951752: CVE-2021-2154 mysql: Server: DML unspecified vulnerability (CPU Apr 2021)"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-20T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "2",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mysql: Server: DML unspecified vulnerability (CPU Apr 2021)",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-04-20T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-2166",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://www.oracle.com/security-alerts/cpuapr2021.html#AppendixMSQL",
						"Description": "https://www.oracle.com/security-alerts/cpuapr2021.html#AppendixMSQL"
					},
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-2166",
						"Description": "CVE-2021-2166"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1951755",
						"Description": "bz#1951755: CVE-2021-2166 mysql: Server: DML unspecified vulnerability (CPU Apr 2021)"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-04-20T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "3",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mysql: InnoDB unspecified vulnerability (CPU Jul 2021)",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-07-20T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-2372",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-2372",
						"Description": "CVE-2021-2372"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1992303",
						"Description": "bz#1992303: CVE-2021-2372 mysql: InnoDB unspecified vulnerability (CPU Jul 2021)"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-07-23T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "4",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mysql: InnoDB unspecified vulnerability (CPU Jul 2021)",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-07-20T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-2389",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-2389",
						"Description": "CVE-2021-2389"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1992309",
						"Description": "bz#1992309: CVE-2021-2389 mysql: InnoDB unspecified vulnerability (CPU Jul 2021)"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-07-23T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "5",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mysql: InnoDB unspecified vulnerability (CPU Oct 2021)",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-10-19T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-35604",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-35604",
						"Description": "CVE-2021-35604"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2016101",
						"Description": "bz#2016101: CVE-2021-35604 mysql: InnoDB unspecified vulnerability (CPU Oct 2021)"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2021-10-20T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "6",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: Crash in get_sort_by_table() in subquery with ORDER BY having outer ref",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-05-10T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-46657",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-46657",
						"Description": "CVE-2021-46657"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2049305",
						"Description": "bz#2049305: CVE-2021-46657 mariadb: Crash in get_sort_by_table() in subquery with ORDER BY having outer ref"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2022-02-01T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "7",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: save_window_function_values triggers an abort during IN subquery",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-05-10T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-46658",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-46658",
						"Description": "CVE-2021-46658"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2049294",
						"Description": "bz#2049294: CVE-2021-46658 mariadb: save_window_function_values triggers an abort during IN subquery"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2022-02-01T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "8",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: Crash in set_var.cc via certain UPDATE queries with nested subqueries",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-05-10T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-46662",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-46662",
						"Description": "CVE-2021-46662"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2050019",
						"Description": "bz#2050019: CVE-2021-46662 mariadb: Crash in set_var.cc via certain UPDATE queries with nested subqueries"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2022-02-03T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "9",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: Crash caused by mishandling of a pushdown from a HAVING clause to a WHERE clause",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-05-10T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-46666",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-46666",
						"Description": "CVE-2021-46666"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2050028",
						"Description": "bz#2050028: CVE-2021-46666 mariadb: Crash caused by mishandling of a pushdown from a HAVING clause to a WHERE clause"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Low"
				}
			},
			"DiscoveryDate": "2022-02-03T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "10",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: Integer overflow in sql_lex.cc integer leading to crash",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2021-08-13T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2021-46667",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2021-46667",
						"Description": "CVE-2021-46667"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2050030",
						"Description": "bz#2050030: CVE-2021-46667 mariadb: Integer overflow in sql_lex.cc integer leading to crash"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-02-03T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "11",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: crash in Used_tables_and_const_cache::used_tables_and_const_cache_join",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2020-05-05T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-27385",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-27385",
						"Description": "CVE-2022-27385"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2075001",
						"Description": "bz#2075001: CVE-2022-27385 mariadb: crash in Used_tables_and_const_cache::used_tables_and_const_cache_join"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-04-13T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "12",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: improper locking due to unreleased lock in the ds_xbstream.cc",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-05-25T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-31621",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-31621",
						"Description": "CVE-2022-31621"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2092353",
						"Description": "bz#2092353: CVE-2022-31621 mariadb: improper locking due to unreleased lock in the ds_xbstream.cc"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-06-01T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		},
		{
			"ordinal": "13",
			"xmlns": "http://www.icasi.org/CVRF/schema/vuln/1.1",
			"notes": {
				"note": {
					"text": "mariadb: DoS due to improper locking due to unreleased lock in plugin/server_audit/server_audit.c",
					"title": "Vulnerability Description",
					"type": "General",
					"ordinal": "1",
					"lang": "en"
				}
			},
			"ReleaseDate": "2022-05-25T00:00:00Z",
			"involvements": {
				"involvement": {
					"party": "Vendor",
					"status": "Completed"
				}
			},
			"CVE": "CVE-2022-31624",
			"remediations": {
				"remediation": {
					"type": "Vendor Fix",
					"description": {
						"text": "\nFor details on how to apply this update, which includes the changes described in this advisory, refer to:\n\nhttps://access.redhat.com/articles/11258\n\nAfter installing this update, the MariaDB server daemon (mysqld) will be restarted automatically.\n    ",
						"lang": "en"
					},
					"URL": "https://access.redhat.com/errata/RHSA-2022:4818"
				}
			},
			"references": {
				"reference": [
					{
						"URL": "https://access.redhat.com/security/cve/CVE-2022-31624",
						"Description": "CVE-2022-31624"
					},
					{
						"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=2092362",
						"Description": "bz#2092362: CVE-2022-31624 mariadb: DoS due to improper locking due to unreleased lock in plugin/server_audit/server_audit.c"
					}
				]
			},
			"threats": {
				"threat": {
					"type": "Impact",
					"Description": "Moderate"
				}
			},
			"DiscoveryDate": "2022-06-01T00:00:00Z",
			"productstatuses": {
				"status": {
					"type": "Fixed",
					"ProductID": [
						"AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4"
					]
				}
			},
			"acknowledgments": {
				"acknowledgment": {
					"Description": ""
				}
			},
			"cvssscoresets": {
				"scoreset": {
					"BaseScore": "",
					"Vector": ""
				}
			}
		}
	],
	"producttree": {
		"xmlns": "http://www.icasi.org/CVRF/schema/prod/1.1",
		"branch": [
			{
				"type": "Product Family",
				"name": "Red Hat Enterprise Linux",
				"branch": [
					{
						"type": "Product Name",
						"name": "Red Hat Enterprise Linux AppStream EUS (v.8.4)",
						"fullproductname": {
							"text": "Red Hat Enterprise Linux AppStream EUS (v.8.4)",
							"productid": "AppStream-8.4.0.Z.EUS",
							"cpe": "cpe:/a:redhat:rhel_eus:8.4::appstream"
						}
					}
				],
				"fullproductname": {}
			},
			{
				"type": "Product Version",
				"name": "mariadb:10.3-8040020220429075504.522a0ee4",
				"fullproductname": {
					"text": "mariadb-10.3-8040020220429075504.522a0ee4",
					"productid": "mariadb:10.3-8040020220429075504.522a0ee4",
					"cpe": "cpe:/a:redhat:rhel_eus:8.4::appstream"
				}
			}
		],
		"relationship": [
			{
				"productreference": "mariadb:10.3-8040020220429075504.522a0ee4",
				"relationtype": "Default Component Of",
				"relatestoproductreference": "AppStream-8.4.0.Z.EUS",
				"fullproductname": {
					"text": "mariadb:10.3-8040020220429075504.522a0ee4 as a component of Red Hat Enterprise Linux AppStream EUS (v.8.4)",
					"productid": "AppStream-8.4.0.Z.EUS:mariadb:10.3-8040020220429075504.522a0ee4",
					"cpe": "cpe:/a:redhat:rhel_eus:8.4::appstream"
				}
			}
		]
	}
}
